recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "widgetsnbextension" %}
-{% set version = "4.0.5" %}
+{% set version = "4.0.10" %}
```

Comment:
Sorry if this is a silly question, but where does this version come from? It's my first time seeing a package like this and I could only find [this](https://github.com/jupyter-widgets/ipywidgets/blob/main/python/widgetsnbextension/package.json#L3), but I don't know if that's correct.

Reply:
Oop! Never mind! I found this!